README.md

```diff
@@ -27,6 +28,30 @@ The current Hilla support has some known limitations:
   is not supported
 * Native image compilation does not work
 
+## Release
+
+To perform a manual relese type the follogin commands.
+Version must be in format N.N.N, for example `1.0.0`.
+Pre-releases can use `alpha`, `beta` and `rc` suffix, followed by a number,
+for example `1.0.0.beta2`.
+
+Environment variables required by the release process:
+
+* JRELEASER_GITHUB_TOKEN: to create release on GitHub
+* JRELEASER_GPG_PUBLIC_KEY, JRELEASER_GPG_SECRET_KEY and
+  JRELEASER_GPG_PASSPHRASE: to sign artifacts
+* JRELEASER_NEXUS2_MAVEN_CENTRAL_USERNAME and
+  JRELEASER_NEXUS2_MAVEN_CENTRAL_PASSWORD: to publish on maven central
+
+Use `-Djreleaser.dry.run=true` flag to test the release without publishing
+artifacts.
+
+```terminal
+mvn clean
+mvn -Pdistribution -Drevision=<version-to-release> -DskipTests -DaltDeploymentRepository=local::file:./target/staging-deploy deploy 
+mvn -N -Pdistribution -Drevision=<version-to-relese> jreleaser:full-release
```

Comment:
workflow has some more parameters, are the just workflow required or also required here?

Reply:
It only has the additional `jreleaser.dry.run` flag, that is documented in the README before the command line example

Reply:
I meant the mvn parameters, but I searched it and now know what they do